### PR TITLE
Adding missing part for WPS 2.0 Rest Binding

### PR DESCRIPTION
**Pull request by gfenoy**

### CODE REVIEW

File: core/openapi/schemas/additionalParameter.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+additionalParameter:
```
Comment (bpross-52n) [inline core/openapi/schemas/additionalParameter.yaml]: Remove top level element

File: core/openapi/schemas/jobInfo.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+jobInfo:
```
Comment (bpross-52n) [inline core/openapi/schemas/jobInfo.yaml]: Remove top level element

File: core/openapi/schemas/statusInfo.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+statusInfo:
```
Comment (bpross-52n) [inline core/openapi/schemas/statusInfo.yaml]: Remove top level element

File: core/openapi/schemas/jobCollection.yaml [hunk 4]
```diff
@@ -1,3 +1,9 @@
-    type: array
-    items:
-      type: string
+jobCollection:
```
Comment (bpross-52n) [inline core/openapi/schemas/jobCollection.yaml]: Remove top level element

Comment (gfenoy) [reply core/openapi/schemas/jobCollection.yaml]: Dear Benjamin, 
I have removed the top level elements.

You can find an exemple of additionalParameters defined for the input identifier "a"  from [here](https://demo.mapmint.com/WPS3/processes/HelloPy2).

**Pull request merged**